qa/tasks/ceph.py

```diff
@@ -1495,9 +1495,8 @@ def healthy(ctx, config):
 
     if ctx.cluster.only(teuthology.is_type('mds', cluster_name)).remotes:
         # Some MDSs exist, wait for them to be healthy
-        ceph_fs = Filesystem(ctx) # TODO: make Filesystem cluster-aware
-        ceph_fs.wait_for_daemons(timeout=300)
-
+        for fs in Filesystem.get_all_fs(ctx):
```

Comment:
Why don't just call `FSStatus.get_filesystems()`? either by importing `FSStatus` or by calling `ceph_fs.status()`.
Any issue in that?